src/Jellyfin.LiveTv/IO/EncodedRecorder.cs

```diff
@@ -73,6 +73,10 @@ private async Task RecordFromFile(MediaSourceInfo mediaSource, string inputFile,
         {
             _targetPath = targetFile;
             Directory.CreateDirectory(Path.GetDirectoryName(targetFile));
+            if (!File.Exists(targetFile))
+            {
+                await File.Create(targetFile).DisposeAsync().ConfigureAwait(false);
```

Comment:
We have a helper to efficiently create new empty files https://github.com/jellyfin/jellyfin/blob/master/src/Jellyfin.Extensions/FileHelper.cs